src/virtual-source-buffer.js

```diff
@@ -515,9 +555,11 @@ export default class VirtualSourceBuffer extends videojs.EventTarget {
    */
   abort() {
     if (this.videoBuffer_) {
+      this.videoBuffer_.updating = true;
```

Comment:
Why is this being added here (and for audio as well)?

According to the spec, when abort is called, if `buffer.updating === true` it should be set to `false` and an `updateend` event should be queued. If the real source buffer behind the wrapper does not have `updating === true` then an `updateend`event will never be fired and the wrapper will forever be `updating === true`